packages/site/cms-site/src/graphQLFetch/fetchInMemoryCache.ts

```diff
@@ -0,0 +1,34 @@
+const cache: Record<string, Response> = {};
+
+type Fetch = typeof fetch;
+
+export function createFetchInMemoryCache(fetch: Fetch): Fetch {
```

Comment:
Should this be part of the public API? Could this be of use in application code somewhere?

Reply:
Maybe.

As a start I wanted to keep the public API to a minimum.